composer.json

```
@@ -21,10 +21,10 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/framework-bundle": "^3.4|^4.0"
+        "symfony/framework-bundle": "^3.4|^4.0|^5.0"
     },
     "require-dev": {
-        "symfony/symfony": "^3.4|^4.0",
+        "symfony/symfony": "^3.4|^4.0|^5.0",
         "symfony/phpunit-bridge": "^4.2",
```

I would update this to 5.0, since it does not conflict with Symfony components
```suggestion
        "symfony/phpunit-bridge": "^5.0",
```